dask_expr/collection.py

```diff
@@ -573,6 +573,11 @@ def dropna(self, how=no_default, subset=None, thresh=no_default):
     def rename(self, columns):
         return new_collection(RenameFrame(self.expr, columns=columns))
 
+    def to_parquet(self, path, **kwargs):
+        from dask_expr.io.parquet import to_parquet
+
+        return to_parquet(self, path, **kwargs)
+
 
 class Series(FrameBase):
     """Series-like Expr Collection"""
```

Comment:
Is the implementation above only on DataFrames?  If so do we need a Series implementation (which I assume is just `to_frame().to_parquet(...)`

Reply:
Ah, ok, that makes sense